packages/calcite-components/src/components/accordion-item/accordion-item.scss

```diff
@@ -10,7 +10,7 @@
  * @prop --calcite-accordion-item-icon-color: defines the icon color for the component.
  * @prop --calcite-accordion-item-expand-icon-color: defines the icon color for the component.
  * @prop --calcite-accordion-item-text-color: defines the text color for the component.
- */
+ **/
```

Comment:
Nitpick: closing with `**/` is not standard in our JSDoc & JSDoc-style comment blocks (see https://stenciljs.com/docs/docs-json#css-variables).